sdk/tables/Azure.Data.Tables/samples/Sample1CreateDeleteTables.md

```diff
@@ -0,0 +1,55 @@
+# Creating and Deleting Tables
+This sample demonstrates how to create and delete a table. To get started, you'll need either a Storage or Cosmos DB account.
+
+## Creating a `TableServiceClient`
+A `TableServiceClient` performs account-level operations like creating and deleting tables. Authentication will be taken care of by creating a `TableSharedKeyCredential` which requires the account name as well as the account key.
+
+You can set `accountName` and `storageAccountKey` based on an environment variable, configuration setting, or any way that works for your application.
```

Comment:
It will be good to add something like.... "For the purpose of this test, we are going to use a Storage Resource" ....
I imagine later we will add information about using Cosmos DB resource too